### DISCUSSION TITLE
Ucapan

### DISCUSSION THREAD

**Discussion by Zettsy** [dewanakl/undangan#19] [General]
Permisi kak, ini saya lagi pakai template undangan nah dibagian ucapan buat hapus/ganti komen yang udah ada gimana ya?

**Comment by dewanakl** [comment]
Kakaknya bisa deploy untuk backend dari api nya sendiri kak, untuk langkah langkahnya ada pada readme undangan ini, terima kasih

**Reply by Zettsy** [reply]
Gak ada cara yang lebih gampang ya kak? Saya gak ngerti,Saya baru banget nyba buat website kayak gini buat tugas sekolah

**Reply by dewanakl** [reply]
Apakah kakanya pernah coba laravel? aplikasi ini mirip dengan laravel kak
